cli/src/cli_services.py

```diff
@@ -33,13 +36,12 @@
 # used as a fall back when required.
 FIREBASE_DEFAULT_RTDB_ID = '{project_id}-default-rtdb'
 
-# The URLs for Firebase RTDBs come in two flavours:
-#
-# DATABASE_NAME.firebaseio.com for databases in us-central1
-# DATABASE_NAME.REGION.firebasedatabase.app for databases in all other locations
-#
-# For now we only support the location of us-central1
-DEFAULT_DB_URL = 'https://{database_id}.firebaseio.com'
+CHECK_CONFIGURATION_MSG = """
+Confirm the correct project has been configured and that the 'init' command has
+been run.  See
+https://github.com/GoogleCloudPlatform/snapshot-debugger/blob/main/README.md for
```

Comment:
Is there a more concise URL?

Reply:
I reduced it to `https://github.com/GoogleCloudPlatform/snapshot-debugger`. With this link the README information will still be displayed on the page. Another option would be to use an actual `shortened` URL, however I'd prefer to keep the actual URL visible.